CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 - Users can customise the error description shown in the Sentry UI by providing an NSDebugDescriptionErrorKey value in the error user info dictionary. (#2120)
 - Add support for dynamic library (#1726)
+- Profile concurrent transactions (#2105)
```

Comment:
`l`: This needs to be updated and moved up. I think we should try to automate the changelog somehow, this is really annoying.

Reply:
Where does it need to be moved to? It looks like it's OK to me 🤔 

But I agree changelog conflicts are annoying. There is a somewhat old thread of automation options [here](https://github.com/olivierlacan/keep-a-changelog/issues/67) and I also have some automation [here](https://github.com/TwoRingSoft/tools/blob/master/bin/migrate-changelog) (and also a thing to extract changelog contents for releases/tags [here](https://github.com/TwoRingSoft/tools/blob/master/bin/changetag)).

Reply:
Ah nm, I see what happened. Not sure how automation can help with that 🤔